vernac/record.ml

```diff
@@ -413,7 +414,7 @@ let build_proj env mib indsp primitive x rp lifted_fields ~poly paramdecls param
 
 (** [declare_projections] prepares the common context for all record
    projections and then calls [build_proj] for each one. *)
-let declare_projections indsp univs ?(kind=Decls.StructureComponent) inhabitant_id flags fieldimpls fields =
+let declare_projections indsp univs ~kind inhabitant_id flags fieldimpls fields =
```

Comment:
Maybe rename kind to proj_kind here too.

Reply:
Not sure about that one, the function is already called `declare_projection`, so I'd find `proj_kind` a bit redundant (and that wouldn't simplify calls since the argument when calling it is called `projection*s*_kind`).